docs/REQUIREMENTS.md

```diff
@@ -16,35 +16,35 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 The following are categorized user stories 
 
 #### Filtering
-- As a user, I want to query a registry for stored artifacts by digest and tag.
-- As a user, I want to query the registry for all stored artifacts that reference another artifact by its tag and digest.
-- As a user, I want to query a registry for all stored artifacts of a particular type that reference another artifact by its tag and digest.
-- As a user, I want to query a registry for all stored artifacts based on annotations that reference another artifact by its tag and digest.
-- As a user, I want to fetch the most up-to-date artifact, collection of artifacts, or application.
-- As an artifact producer, I would like to reduce the number of tags that reference artifacts.
+1. As a user, I want to query a registry for stored artifacts by digest and tag.
+1. As a user, I want to query the registry for all stored artifacts that reference another artifact by its tag and digest.
+1. As a user, I want to query a registry for all stored artifacts of a particular type that reference another artifact by its tag and digest.
+1. As a user, I want to query a registry for all stored artifacts based on annotations that reference another artifact by its tag and digest.
+1. As a user, I want to fetch the most up-to-date artifact, collection of artifacts, or application.
+1. As an artifact producer, I would like to reduce the number of tags that reference artifacts.
```

Comment:
"I'm number 1!"

Reply:
Would you prefer that the Markdown numbers them explicitly? It ends up getting rendered as incrementing numbers either way.

Reply:
oh neat! didnt know that